### PR TITLE
VHSchan support

### PR DESCRIPTION
**Pull request by MrPurple666**

### CODE REVIEW

File: Kuroba/app/src/test/java/com/github/k1rakishou/chan/core/site/parser/ReplyParserTest.kt [hunk 1]
```diff
@@ -94,6 +95,18 @@ class ReplyParserTest {
     assertEquals(28950, (replies[1] as ReplyParser.ExtractedQuote.Quote).postId)
   }
 
+@Test
+  fun `test extract quotes from vhschan_org comment`() {
+    whenever(siteManager.bySiteDescriptor(any())).thenReturn(Vhschan())
+
+    val replies = replyParser.extractCommentReplies(SiteDescriptor.create("vhschan.org>
```
Comment (K1rakishou) [inline Kuroba/app/src/test/java/com/github/k1rakishou/chan/core/site/parser/ReplyParserTest.kt]: It seems like something wrong went here.

**Comment by K1rakishou** [comment]
The favicon doesn't work (returns 404). Some endpoints were incorrect (apparently) so I changed them. Some thumbnails do not show up (404). If this is fine then I can merge it otherwise you can try to fix it.

**Comment by MrPurple666** [comment]
> The favicon doesn't work (returns 404). Some endpoints were incorrect (apparently) so I changed them. Some thumbnails do not show up (404). If this is fine then I can merge it otherwise you can try to fix it.

I fixed favicon, renamed some things to avoid problems as many were with a upercase "V" so everything is in lowercase, boards name and description have been fixed! Probably the issues with 404 were also fixed with these changes.

**Comment by MrPurple666** [comment]
Sorry for force push, my git setup was wrong and the commits "were not" my own (bcz wrong email). It will not happen again.

**Review by K1rakishou — APPROVED** [review]
LGTM

**Pull request merged**